MODULE.bazel.lock

```diff
@@ -2599,28 +2627,69 @@
         ]
       }
     },
-    "@@rules_jvm_external~//:non-module-deps.bzl%non_module_deps": {
+    "@@rules_kotlin~//src/main/starlark/core/repositories:bzlmod_setup.bzl%rules_kotlin_extensions": {
```

Comment:
I don't know what any of the changes from here on area ll about? @rockwotj ?

Reply:
They are transitive dependencies from protobuf, which now supports kotlin protos in the bazel build. It's fine